outputs.tf

```
@@ -133,6 +133,11 @@ output "database_subnet_group" {
   value       = concat(aws_db_subnet_group.database.*.id, [""])[0]
 }
 
+output "database_subnet_group_name" {
+  description = "Name of database subnet group"
+  value       = aws_db_subnet_group.database.*.name[0]
```

lets keep it consistent:
```suggestion
  value       = concat(aws_db_subnet_group.database.*.name, [""])[0]
```